distributions/otelcol-k8s/manifest.yaml

```diff
@@ -0,0 +1,71 @@
+dist:
+  module: github.com/open-telemetry/opentelemetry-collector-releases/k8s
+  name: otelcol-k8s
+  description: OpenTelemetry Collector for Kubernetes
+  version: 0.96.0
+  output_path: ./_build
+  otelcol_version: 0.96.0
+
+extensions:
+  - gomod: go.opentelemetry.io/collector/extension/zpagesextension v0.96.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/extension/basicauthextension v0.96.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/extension/bearertokenauthextension v0.96.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/extension/headerssetterextension v0.96.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/extension/healthcheckextension v0.96.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/extension/httpforwarderextension v0.96.0
```

Comment:
Do you have a specific use case in mind for this one?

Reply:
Not specifically.  I could remove it if we want to keep slimming down the manifest.